remediation/fence-agents-remediation/how-it-works.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: How It Works
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 1
+---
+
+## How Fence Agents Remediation works?
+
+Fence Agents Remediation (*FAR*) operator uses a management interface or a traditional API to remediate unhealthy kubernetes nodes.
+It was tested with [Node HealthCheck Operator](https://github.com/medik8s/node-healthcheck-operator) (NHC), that creates a Custom Resource (CR) for FAR with information about the unhealthy Node.
+
+FAR remediates a node by rebooting an unhealthy node, and moving any remaining workloads from the node to other nodes, so they can be processed in other nodes and be isolated from the unhealthy node. The reboot is done by executing a fence agent for the unhealthy node, while evicting the workloads from this node is achieved by tainting the node and deleting the workloads.
```

Comment:
"and moving any remaining workloads from the node to other nodes, so they can be processed in other nodes and be isolated from the unhealthy node."

maybe a bit less "nodes" are good enough?
-> "and evicting workloads from the unhealthy node."